### DISCUSSION TITLE
SpiralLinux "Barebone"?

### DISCUSSION THREAD

**Discussion by marwenius** [SpiralLinux/SpiralLinux-project#499] [General]
Is it possible to get a “Barebone” version of SpiralLinux? Either by manually customizing the “Builder” version? Which files need to be adapted and how (on the installed system or on the live build base)?

Or could you provide a corresponding live build base so that one can build it themselves?

By “Barebone” I mean a headless / server-like version without a GUI, so that I start directly in the terminal after logging in and can assemble my own desktop environment from scratch.

**Comment by geckolinux** [comment]
Hi there, that's exactly the purpose of the "Builder" edition. It has IceWM to provide just enough of a GUI to run Firefox and Synaptic so you can then install and customize your desktop environment of choice.

**Comment by Cinnibars** [comment]
Hi, I hope it's okay if I piggyback onto this thread as my question is very similar. As a new Linux user I'm really happy with the simple setup and I'm hoping to use Spiral Linux to set up my headless media storage/server machine.

So, starting with the barebone Builder edition... rather than proceeding to customize a DE, is it possible to remove the desktop elements entirely so I'm left with a headless Debian? I'd like to benefit from the customizations like media codecs and easy snapshotting but I plan to only access the GUI via SSH if at all.

**Reply by geckolinux** [reply]
Hi there @Cinnibars, SpiralLinux doesn't sound like the best choice for that application. The Builder edition is not intended to be used as a minimal installation to later *remove* elements, but rather as a base with all of the complicated stuff already in place for later *adding* more desktop functionality. For what you're trying to do there are several excellent Linux distros that are tailor made for storage or media with an excellent admin interface accesible via a web browser. For that you might want to look at https://openmediavault.org or https://www.truenas.com/truenas-scale .

**Comment by Cinnibars** [comment]
Thank you for the response! I agree, those will  make my life a lot easier while I'm still learning. Much like Spiral Linux has done :)
